features/minion/repository/pom.xml

```diff
@@ -74,6 +74,7 @@
                                 <feature>dominion-grpc-client</feature>
                                 <feature>dominion-secure-credentials-vault</feature>
                                 <featue>minion-rest-service</featue>
+                                <feature>minion-health-check</feature>
```

Comment:
Also need to add dependency below for <artifactId>health-check</artifactId>

Reply:
done